app/views/user/update_profile.html.slim

```diff
@@ -50,39 +50,40 @@
       -else
         =f.hidden_field :website, value: "NOTOWNER"
         =f.hidden_field :about, value: "NOTOWNER"
-      tr  
+      tr
         th =f.label :preferred_locale, t('.interface_language'), class: 'collection-label'
         td =f.select :preferred_locale, options_for_select(@dropdown_locales.map{|code| [I18n.translate(code), code]}, @user.preferred_locale), :include_blank => true
-      tr  
+      tr
         td(colspan="2")
           =label_tag :dictation_language, t('.default_dictation_language'), class: 'collection-label'
-      tr  
+      tr
         td(colspan="2")
           =select_tag :dictation_language, {}, {onchange:'updateCountry()', class: 'w30'}
           |&nbsp;
           =select_tag :dialect, {}, {class: 'w40'}
 
-    p.email_notifications =t('.select_notification')
-    table.form.email_notifications
-      tr
-        td =f.check_box("notifications[user_activity]", {checked: @user.notification.user_activity})
-        th =f.label :user_activity, t('.user_activity')
-      -if @user.owner
+    =f.fields_for :notification do |nf|
```

Comment:
We can do fields_for, this is so we don't have to manually add the names. This will fall under `notification_attributes` and we can pass in `@user.update(user_params)` and the notification will also be updated